index.html

```diff
@@ -13350,7 +13350,7 @@ <h3>States and Properties</h3>
 					<li>Correct usage with regard to <abbr title="Document Object Model">DOM</abbr> tree structure, such as an <a>element</a> being owned by more than one other element.</li>
 					<li>Elements with <abbr title="Accessible Rich Internet Application">WAI-ARIA</abbr> <a>roles</a> correctly implement the  behavior of the specified role. For example, user agents do not verify that an element with a role of <rref>checkbox</rref> actually behaves like a checkbox.</li>
 					<li>Elements that do not correctly observe required child / parent role relationships or that appear elsewhere than in their required parent.</li>
-					<li>Determining whether <pref>aria-activedescendant</pref> actually points to a <abbr title="Document Object Model">DOM</abbr> descendant or owned element.</li>
+					<li>Determining whether <pref>aria-activedescendant</pref> actually points to a <a>owned</a> element of the container widget.</li>
```

Comment:
Change "a owned" to "an owned".
```suggestion
					<li>Determining whether <pref>aria-activedescendant</pref> actually points to an <a>owned</a> element of the container widget.</li>
```